pkg/render/applicationlayer/applicationlayer.go

```diff
@@ -311,6 +329,10 @@ func (c *component) containers() []corev1.Container {
 			)
 		}
 
+		if c.config.ALPEnabled {
```

Comment:
Same here - should probably be `PerHostALPEnabled` or `AnyALPEnabled`, for clarity (with field level comments in the struct).